src/client/components/MyArea/MyArea.module.scss

```diff
@@ -46,6 +46,7 @@
 .Header {
   display: flex;
   height: 6.4rem;
+  min-height: 6.4rem;
```

Comment:
Waar heb je deze voor nodig?

Reply:
anders verspringt de header bij het aanroepen van de zoekfunctie op mijn buurt

Reply:
Ok, wat is de oorzaak van dat verspringen?

Reply:
Omdat de height initieel lager lijkt te zijn dan 6.4

Reply:
GheGhe maar... waarom verschuift de height. Als de zoekfunctie dicht is ziet het er toch goed uit?